package.json

```diff
@@ -45,9 +45,15 @@
   ],
   "license": "BSD-3-Clause",
   "dependencies": {
+    "@types/adm-zip": "^0.4.32",
+    "adm-zip": "^0.4.13",
```

Comment:
i wish there were a better way to "optional" this - but as Z mentioned we might want to break the actual deps that involve zip/fetch into a "breakout" part of this so it can at least be tree-shook out

Reply:
@gnarf see the yarn workspaces I've added. What do you think about that file structure?

Reply:
Monorepo structure tabled for now